src/utils/rxUtils/handleRxProps.js

```diff
@@ -0,0 +1,54 @@
+import { Observable } from 'rxjs/Observable';
+import dispatch from '../dispatch';
+import camelize from '../camelize';
+import warn from '../warn';
+
+export default function handleRxProps({ fieldProps, fields, form }) {
```

Comment:
This name is little bit confusing for me...
`handleRxProps` doesnt really says what is causing this function to be called as opposed to 
`handleButtonClick`

Reply:
Agree. This function needs more descriptive name. Will do as the last step of this pull request, after all the major questions are resolved. Thanks!